### ISSUE TITLE
HTML "select" tags have too much indentation

### ISSUE THREAD

**Issue by knocte** [beautify-web/js-beautify#1097]
HTML "select" tags have too much indentation.

# Input
The code looked like this before beautification:
```
<select class="col-4 form-input" ng-model="info.dateRange" ng-init="info.dateRange = 'last24'">
        <option selected="selected" value="last24">{{"dashboard_records_text_7" | translate}}</option>
        <option value="last7">{{"dashboard_records_text_8" | translate}}</option>
        <option value="custom">{{"dashboard_records_text_9" | translate}}</option>
    </select>
```

# Expected Output
The code should have looked like this after beautification:
```
<select class="col-4 form-input" ng-model="info.dateRange" ng-init="info.dateRange = 'last24'">
    <option selected="selected" value="last24">{{"dashboard_records_text_7" | translate}}</option>
    <option value="last7">{{"dashboard_records_text_8" | translate}}</option>
    <option value="custom">{{"dashboard_records_text_9" | translate}}</option>
</select>
```

# Actual Output
The  code actually looked like this after beautification:
```
<select class="col-4 form-input" ng-model="info.dateRange" ng-init="info.dateRange = 'last24'">
        <option selected="selected" value="last24">{{"dashboard_records_text_7" | translate}}</option>
        <option value="last7">{{"dashboard_records_text_8" | translate}}</option>
        <option value="custom">{{"dashboard_records_text_9" | translate}}</option>
    </select>
```

## Environment
OS: Mac
Version tested with: 1.6.7

## Settings
Example:
```json
var options = {
    "wrap_line_length": 80,
    "indent_handlebars": true
};
```

**Comment by HookyQR** [comment]
`select` is an unformatted tag by default. This isn't a bug. Add the following to your options:
```javascript
	"unformatted": ["a", "abbr", "area", "audio", "b", "bdi", "bdo", "br", "button", "canvas", "cite", "code", "data",
		"datalist", "del", "dfn", "em", "embed", "i", "iframe", "img", "input", "ins", "kbd", "keygen", "label", "map",
		"mark", "math", "meter", "noscript", "object", "output", "progress", "q", "ruby", "s", "samp", "small", "span",
		"strong", "sub", "sup", "svg", "template", "textarea", "time", "u", "var", "video", "wbr", "text", "acronym",
		"address", "big", "dt", "ins", "strike", "tt"]
```

**Comment by Oxicode** [comment]
+1

**Comment by bitwiseman** [comment]
Fixed in 1.8.0-rc4 using the new `inline` setting. 
@HookyQR 
Unformatted no longer needed.

**Comment by HookyQR** [comment]
Any timeline for a 1.8.0 release. Would love to update the VS Code extension, but waiting for the `-rc` to drop off.
